services/QuillCMS/spec/workers/rematch_response_worker_spec.rb

```diff
@@ -128,16 +129,19 @@
         create(:response, params)
       end
     end
+    let(:options) { { 'fire_pusher_alert' => true, 'question_key' => 'some_question_key'} }
```

Comment:
Minor suggestion, but I'd be tempted to have an earlier `let(:question_key) { 'some_question_key}` and then have `'question_key' => question_key` here.  That way you can refer to `question_key` as a variable in your expectation below rather than typing the string literal again and risk that getting messed up somehow.